clmm/theory/func_layer.py

```diff
@@ -61,9 +61,10 @@ def compute_3d_density(r3d, mdelta, cdelta, z_cl, cosmo, delta_mdef=200,
             * 'critical'
             * 'virial'
 
-    alpha_ein : float, optional
-        If `halo_profile_model=='einasto'`, set the value of the Einasto slope. Option only
-        available for the NumCosmo backend
+    alpha_ein : float, 'cosmo'(CCL backend only), optional
+        If `halo_profile_model=='einasto'`, set the value of the Einasto slope.
+        Option only available for the NumCosmo and CCL backends.
+        Set to 'cosmo' to use the cosmology-dependent :math:`\alpha` parameter in the CCL backend.
```

Comment:
I find it counter-intuitive that alpha could also be the string `cosmo` for the CCL backend. I wonder whether we should simply mention that not specifying alpha means using the backend default value, which is not the same for CCL or NumCosmo. For CCL this is the cosmology-dependent value and for NumCosmo it is alpha=0.25. 

What do you think?

Reply:
Actually, trying to implement what I am suggesting, I realise this is actually trickier than it seems as CCL cannot update a parameter to a `None` value (does not give an error, but does not take it into account it seems). If we first call CCL with a user-defined value for alpha, putting back `alpha_ein=None` in the function call does not revert to the CCL default value based on cosmology. This is OK though if calling the function with `alpha_ein=cosmo, as @hsinfan1996 implemented.

I still think it would be good to have a more intuitive behaviour, but we maybe can address this in a new issue?